ibu-imager/ops/ops.go

```diff
@@ -139,24 +138,24 @@ func (o *ops) waitForEtcd(healthzEndpoint string) error {
 	}
 }
 
-func (o *ops) RunRecert(recertContainerImage, authFile string, additionalArgs ...string) error {
+func (o *ops) RunRecert(recertContainerImage, authFile, recertConfigFile string, additionalPodmanParams ...string) error {
+
 	command := "podman"
-	args := append(clusterconfig.PodmanRecertArgs, authFile, "--name", "recert",
+	args := append(podmanRecertArgs, "--name", "recert",
 		"-v", "/etc:/host-etc",
 		"-v", "/etc/kubernetes:/kubernetes",
 		"-v", "/var/lib/kubelet:/kubelet",
-		"-v", common.BackupCertsDir+":/certs",
+		"-v", "/var/tmp:/var/tmp",
 		"-v", "/etc/machine-config-daemon:/machine-config-daemon",
-		recertContainerImage,
-		"--etcd-endpoint", "localhost:2379",
-		"--static-dir", "/kubernetes",
-		"--static-dir", "/kubelet",
-		"--static-dir", "/machine-config-daemon")
-
-	// Add additional arguments to the command
-	args = append(args, additionalArgs...)
+		"-e", fmt.Sprintf("RECERT_CONFIG=%s", recertConfigFile),
+	)
+	if authFile != "" {
+		args = append(args, "--authfile", authFile)
+	}
 
-	_, err := o.RunInHostNamespace(command, args...)
+	args = append(args, additionalPodmanParams...)
+	args = append(args, recertContainerImage)
+	_, err := o.hostCommandsExecutor.Execute(command, args...)
```

Comment:
```suggestion
	if _, err := o.hostCommandsExecutor.Execute(command, args...); err != nil {
		return fmt.Errorf("failed to run recert tool container: %w", err)
	}
```

Reply:
Why we should change all those places? Do you have some link to why this way is better?

Reply:
See https://github.com/openshift-kni/lifecycle-agent/pull/101#discussion_r1400597576